packages/devtools_app/lib/src/performance/flutter_frames_chart.dart

```diff
@@ -106,18 +108,21 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
         right: denseSpacing,
         bottom: defaultSpacing,
       ),
-      height: defaultChartHeight,
+      height: defaultChartHeight + scrollBarOffset,
```

Comment:
nit: maybe move scrollBarOffset to a util file so we remember to use the same const everywhere we need non-overlapping scrollbars?

Reply:
done